WordPress/Classes/ViewRelated/Jetpack/Branding/Fullscreen Overlay/JetpackFullscreenOverlaySiteCreationViewModel.swift

```diff
@@ -6,6 +6,9 @@ struct JetpackFullscreenOverlaySiteCreationViewModel: JetpackFullscreenOverlayVi
 
     let phase: JetpackFeaturesRemovalCoordinator.SiteCreationPhase
     let source: String
+    let actionInfoText: NSAttributedString? = nil
+    let footnote: String? = nil
+    let learnMoreButtonURL: String? = nil
```

Comment:
I'm not sure which one this button is. Is this for the terms & conditions? Assuming this is still in progress, perhaps this will be clearer in your next PRs.